### DISCUSSION TITLE
Modification - Total row for table

### DISCUSSION THREAD

**Discussion by Devaney** [Legal-and-General/canopy#876] [Canopy requests]
Can we add a total row to the table component?
Would it be best to add it as a `tfoot` element e.g. `lg-table-foot` ?

Desktop example
![image](https://user-images.githubusercontent.com/7091410/169808707-c04ed1db-f8ce-463a-8afd-349a72595e97.png)

Mobile Example
![image](https://user-images.githubusercontent.com/7091410/169808785-bd79233c-68e7-4e7b-b44a-71052663a321.png)

**Comment by elenagarrone** [comment]
@Devaney seems like a good thing to add 👍 

As per current implementation of the table head I would use the `tfoot` HTML tag and then add the component with a selector of `lg-table-footer`.

**Reply by alexcanning** [reply]
Looks good to me.

**Comment by elenagarrone** [comment]
@Devaney feel free to create the PR whenever you have time.
